catkin_home/src/action_selectors/scripts/hear.py

```diff
@@ -36,6 +32,12 @@
 NUM_PROCESSES_BEAM_SEARCH = 2
 
 
+if FORCE_USE_STT != "online":
+    # To avoid having to install DS2's dependencies and 
+    # all the required files if using this only online.
+    from DeepSpeech.deploy.ros_server import ASRServer
```

Comment:
How this node receives the requests of STT? Where does it assign the value to FORCE_USE_STT depending on internet connection?

Reply:
Specifically, `FORCE_USE_STT` is a constant (flag) that when set, this node never check for internet connection and everytime it directly goes to the set option.